### PR TITLE
Clarify that `--gzip` supports using `--nsTo` and `--nsFrom` options. 

### PR DESCRIPTION
**Pull request by MongoCaleb**
## DESCRIPTION
Clarify that `--gzip` supports using `--nsTo` and `--nsFrom` options. 

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/database-tools/docsworker-xlarge/DOCSP-34697/mongorestore/

## JIRA
https://jira.mongodb.org/browse/DOCSP-34697

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=656f60e87a5cb5fef517e445

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Comment by MongoCaleb** [comment]
External review completed by Felicia Shieh in slack.

**Review by jason-price-mongodb — APPROVED** [review]
@MongoCaleb LGTM with suggested adjustments. Cheers.

File: source/includes/note-use-nsfrom-nsto.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You can use the :option:`--nsFrom <mongorestore --nsFrom>` and
```
[the remark this answers is not in the document]
Comment (MongoCaleb) [reply source/includes/note-use-nsfrom-nsto.rst]: @jason-price-mongodb your suggestion made me realize that different wording was warranted in each place, so I removed the include and rewrote. Please LMK what you think.

**Review by jason-price-mongodb — APPROVED** [review]
@MongoCaleb looks ok. Cheers!

**Pull request merged**